text/0256-react-cdk-add-jsx-tsx-support.md

```diff
@@ -0,0 +1,182 @@
+---
+feature name: react-cdk-add-jsx-tsx-support
+start date: 2020-09-27
+rfc pr:
+related issue: https://github.com/aws/aws-cdk-rfcs/issues/256
+---
+
+# Summary
+
+Support the expression of cloud infrastructure resources via React JSX / TSX. Cloud solutions
+are composed of many services. Being able to efficiently and naturally express the composition
+is a developer ergonomics/DX concern.
+
+# README
+
+Create CDK applications using React.
+
+## Hello ReactCDK
+
+Example ReactCDK application
+
+```jsx
+const { ReactCDK, App, Stack } = require('@aws-cdk/react')
+const Bucket = require('@aws-cdk/react/s3')
+const { CloudFrontDistribution, Origins, DefaultOrigin } = require('@aws-cdk/react/cloudfront')
+const { Api, Resource, Integration } = require('@aws-cdk/react/apigateway')
+const Lambda = require('@aws-cdk/react/lambda')
+
+const EchoLambda = (
+  <Lambda>
+  {
+    async ({event, context}) => ({
+      "statusCode": 200,
+      "headers": {
+            "Content-Type": "application/json"
+        }
+      "body": JSON.stringify(event)
+    })
+  }
+  </Lambda>
+)
+
+const EchoApi = (
+  <Api>
+    <Resource path="*">
+      <Integration type="lambda">
+        <EchoLambda />
+      </Integration>
+    </Resource>
+  </Api>
+)
+
+const WebsiteBucket = () => (
+  <Bucket src="./public" />
+)
+
+const MyCloudFrontDistribution = () => (
+  <CloudFrontDistribution>
+    <Origins>
+      <DefaultOrigin oai="true">
+        <WebsiteBucket />
+      </DefaultOrigin>
+      <Origin path="/api/*">
+        <EchoApi />
+      </Origin>
+    </Origins>
+  </CloudFrontDistribution>
+)
+
+const MyApp = (
+<App>
+  <Stack>
+    <MyCloudFrontDistribution />
+  </Stack>
+</App>
+)
+
+ReactCDK.render(MyApp, {region: 'us-east-1'})
+```
+
```

Comment:
Can you add to the README what would be the next steps users need to take in order to synth and deploy this app?